pom.xml

```diff
@@ -166,6 +167,7 @@
                 <global_kafka_version>${base_kafka_version}.${hdp_version}-${build_number}</global_kafka_version>
                 <global_hbase_version>1.1.1</global_hbase_version>
                 <global_hbase_guava_version>12.0</global_hbase_guava_version>
+                <global_solr_version>7.4.0</global_solr_version>
```

Comment:
The Solr version for HDP-2.5.0.0 should also stay as 6.6.2 to match the previous behavior.

Reply:
See my other comment.